packages/devtools_app/lib/src/screens/memory/panes/allocation_profile/allocation_profile_table_view.dart

```diff
@@ -412,3 +407,36 @@ class _RefreshOnGCToggleButton extends StatelessWidget {
     );
   }
 }
+
+class _ProfileHelpLink extends StatelessWidget {
+  const _ProfileHelpLink({Key? key}) : super(key: key);
+
+  static const _documentationTopic = 'allocationProfile';
+
+  @override
+  Widget build(BuildContext context) {
+    return HelpButtonWithDialog(
+      gaScreen: analytics_constants.memory,
+      gaSelection:
+          analytics_constants.topicDocumentationButton(_documentationTopic),
+      dialogTitle: 'Memory Allocation Profile Help',
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.end,
+        children: [
+          const Text(
+              'The allocation profile tab displays information about currently\n'
+              'allocated objects in the Dart heap of the currently selected\n'
+              'isolate.'),
```

Comment:
I think you can lose both "current"s. If you prefer to keep one of them, I'd replace "selected" with "current".

```suggestion
              'The allocation profile tab displays information about\n'
              'allocated objects in the Dart heap of the selected\n'
              'isolate.'),
```